requirements/test.txt

```diff
@@ -6,3 +6,4 @@ pytest-ordering
 pytest-timeout
 xarray
 zarr
+scikit-image
```

Comment:
Could you also add scikit-image to `requirements/docs.txt`?

Reply:
done